bin/lib/translation-api-limit.js

```diff
@@ -35,7 +35,7 @@ function monthCreate({ month, year }) {
 	});
 }
 
-function withinApiLimit({ provider }) {
+function withinApiLimit(provider) {
```

Comment:
similarly, how do we handle the dashboard when deepl is within limit & google isn't but both have been requested?